### DISCUSSION TITLE
There are some error when i build jaxlib from source

### DISCUSSION THREAD

**Discussion by OptNuo** [jax-ml/jax#18037] [Q&A]
### Description

python3.9.5
bazel6.1.2
numpy1.26.0

I execute ：python build/build.py --enable_cuda 

BUG：
ERROR: /home/nkn/.cache/bazel/_bazel_nkn/b4d615bfc19ac426847a2ea815c61f11/external/local_config_python/BUILD:78:8: in outs attribute of genrule rule @local_config_python//:python_include: Genrules without outputs don't make sense
ERROR: /home/nkn/.cache/bazel/_bazel_nkn/b4d615bfc19ac426847a2ea815c61f11/external/local_config_python/BUILD:78:8: Analysis of target '@local_config_python//:python_include' failed

How can i solve it？

### What jax/jaxlib version are you using?

jaxlib 0.4.16

### Which accelerator(s) are you using?

GPU

### Additional system info

WSL ubuntu20.04

### NVIDIA GPU info

CUDA11.8   cudnn8.9.5

**Comment by hawkinsp** [comment]
See also https://github.com/google/jax/issues/18036

No need for both a discussion and an issue. Let's follow up in the issue.
